packages/blocks/README.md

```diff
@@ -492,6 +512,36 @@ _Returns_
 
 -   `Array`: A list of blocks.
 
+### registerBlockBindingsSource
+
+Registers a new block bindings source with an object defining its behavior. Once registered, the source is available to be connected to the supported block attributes.
+
+_Usage_
+
+```js
+import { _x } from '@wordpress/i18n';
+import { registerBlockBindingsSource } from '@wordpress/blocks';
+
+registerBlockBindingsSource( {
+	name: 'plugin/my-custom-source',
+	label: _x( 'My Custom Source', 'block bindings source' ),
+	usesContext: [ 'postType' ],
+	getValues: () => getSourceValues(),
+	setValues: () => updateMyCustomValuesInBatch(),
+	canUserEditValue: () => true,
+} );
+```
+
+_Parameters_
+
+-   _source_ `Object`: Properties of the source to be registered.
```

Comment:
Are all of the properties mandatory?

Reply:
No, they are not. I wrapped the optional ones with `[]` because I thought that was the way to do it. Should I clarify it in the description anyway?

Reply:
Ah, I missed that nuance. I would welcome the update to the description of the parameters to make it easier to discover that some params are optional.

Reply:
Added in this commit: https://github.com/WordPress/gutenberg/pull/65713/commits/a5792ef501c8945d5f0b2537cb2d8a4771606d18